go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/projectcontour/contour
 
 go 1.19
 
+// remove once https://github.com/cert-manager/cert-manager/issues/5953 is fixed
+replace github.com/Venafi/vcert/v4 => github.com/jetstack/vcert/v4 v4.9.6-0.20230127103832-3aa3dfd6613d
```

Comment:
huh never end up running `go list ...` very often but confirmed this is an issue